renormalizer/mps/mpo.py

```diff
@@ -554,6 +558,69 @@ def ph_onsite(cls, mol_list: MolList, opera: str, mol_idx:int, ph_idx=0):
         mpo.build_empty_qn()
         return mpo
 
+    @classmethod
```

Comment:
Still, I don't think this method works for scheme 4. If it is currently not necessary at least we can check the `mol_list` to ensure scheme 4 is not used.

Reply:
And hopefully this method can take the place of `displacement`. I think I can handle this in the near future.

Reply:
What is displacement for ?

Reply:
Similar to this method but somehow limited. Used when creating flux operators.

Reply:
I see.

Reply:
I add a `mol_list` check in this function. Please see the new update.